### ISSUE TITLE
TypeError in master I think

### ISSUE THREAD

**Issue by jmchilton** [galaxyproject/planemo#782]
```
galaxy.tools.deps.conda_util: ERROR: Failed to execute command: /home/travis/miniconda3/bin/conda build /home/travis/build/galaxyproject/planemo/project_templates/conda_answers/exercise_2/fleeqtk
Traceback (most recent call last):
  File "/home/travis/build/galaxyproject/planemo/.tox/py27/lib/python2.7/site-packages/galaxy/tools/deps/conda_util.py", line 227, in exec_command
    return self.shell_exec(cmd, env=env)
  File "/home/travis/build/galaxyproject/planemo/planemo/io.py", line 47, in shell
    info(cmds)
  File "/home/travis/build/galaxyproject/planemo/planemo/io.py", line 54, in info
    click.echo(click.style(message, bold=True, fg='green'))
  File "/home/travis/build/galaxyproject/planemo/.tox/py27/lib/python2.7/site-packages/click/termui.py", line 393, in style
    return ''.join(bits)
TypeError: sequence item 2: expected string, list found
```

Recent breakage may be due to galaxy-lib update?

**Comment by jmchilton** [comment]
Maybe this is not in master and I was not staring at the CI report I thought - silly me :(.

**Comment by nsoranzo** [comment]
That commit is a bit broken, I'm testing a fix.

**Comment by nsoranzo** [comment]
Fixed in #774.
